### ISSUE TITLE
queryByText fails when component with icon is rendered

### ISSUE THREAD

**Issue by mgenov** [callstack/react-native-testing-library#142]
## Versions
```jsx
"react": "16.8.6",
"react-dom": "16.8.6",
"react-native": "0.58.4",
"react-native-testing-library": "1.6.1",
"react-native-vector-icons": "6.0.2",
```

## Description
The queryByText call fails when is rendered component with venctor icons:
```
 Cannot read property 'toString' of null
```
## Reproducible Demo

```jsx
import OctIcon from 'react-native-vector-icons/Octicons'
import { render } from 'react-native-testing-library'

it('crashes with error', () => {
    const { queryByText } = render(
      <View>
        <Text>Hello</Text>
        <OctIcon  name={'check'} />
      </View>
    )    
    expect(queryByText('Hello')).toBeTruthy()
})

```
-->

**Comment by jakemmarsh** [comment]
Same issue here. Looks like it's been fixed in #136 but not released.

**Comment by thymikee** [comment]
Released 1.6.2, thanks for a reminder @jakemmarsh :)
